test/commands/call.test.ts

```diff
@@ -108,4 +110,23 @@ describe('apify call', () => {
         expect(EXPECTED_INPUT).toStrictEqual(input!.value);
         expect(EXPECTED_INPUT_CONTENT_TYPE).toStrictEqual(input!.contentType);
     });
+
+    it('should work with just the actor name', async () => {
+        try {
+            await CallCommand.run([ACTOR_NAME], import.meta.url);
+        } catch (err) {
+            expect(err).toBeFalsy();
+        }
```

Comment:
i never liked this approach for testing that something does not fail. 

better use `expect.resolves`